src/utils/pad.ts

```diff
@@ -0,0 +1,9 @@
+const pad = (number: number, length: number) => {
```

Comment:
JS has built in pad now:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart